python/servo/command_base.py

```diff
@@ -582,41 +571,6 @@ def handle_android_target(self, target):
             return True
         return False
 
-    def ensure_bootstrapped(self, target=None):
-        if self.context.bootstrapped:
-            return
-
-        target_platform = target or host_triple()
-
-        rust_root = self.config["tools"]["rust-root"]
-        rustc_path = path.join(
-            rust_root, "rustc", "bin", "rustc" + BIN_SUFFIX
-        )
-        rustc_binary_exists = path.exists(rustc_path)
-
-        base_target_path = path.join(rust_root, "rustc", "lib", "rustlib")
-
-        target_path = path.join(base_target_path, target_platform)
-        target_exists = path.exists(target_path)
-
-        # Always check if all needed MSVC dependencies are installed
-        if "msvc" in target_platform:
-            Registrar.dispatch("bootstrap", context=self.context)
```

Comment:
Given the appveyor failure (`thread 'main' panicked at 'OpenSSL library directory does not exist: C:\projects\servo\.servo\msvc-dependencies\openssl\1.1.0e-vs2015\lib64', C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\openssl-sys-0.9.22\build.rs:69:8`), I fear we may have lost something important by removing this line.

Reply:
Reverted.